tests/test_python.py

```diff
@@ -45,3 +96,127 @@ def test_pip(auto_container):
+    proc = container_per_test.connection.process.filter(comm="python3")
+
+    assert len(proc) > 0  # not empty process list
+
+    x = None
+
+    for p in proc:
+        x = p.args
+        if "http.server" in x:
+            break
+
+    # checks expected parameter of the running python process
+    assert "http.server" in x, "http.server not running."
+
+    # checks that the expected port is listening in the container
+    assert container_per_test.connection.socket(
+        "tcp://0.0.0.0:" + port
+    ).is_listening
+
+
+@pytest.mark.parametrize(
+    "container_per_test", CONTAINER_IMAGES_T, indirect=["container_per_test"]
+)
+def test_python_webserver_2(container_per_test, host, container_runtime):
+    """Test python wget library able to get remote files"""
+
+    # ID of the running container under test
+    c_id = container_per_test.container_id
+
+    destdir = bcdir + outdir
+
+    appl2 = "communication_examples.py"
+
+    url = "https://www.suse.com/assets/img/suse-white-logo-green.svg"
+
+    xfilename = "suse-white-logo-green.svg"
+
+    # install wget for python
+    container_per_test.connection.run_expect([0], "pip install wget")
+
+    # copy an application file from the local test-server into the running Container under test
+    host.run_expect(
+        [0],
+        f"{container_runtime.runner_binary} cp {orig + appdir + appl2} {c_id}:{bcdir + appdir}",
+    )
+
+    # check the test python module is present in the container
+    assert container_per_test.connection.file(bcdir + appdir + appl2).is_file
+
+    # check expected file not present yet in the destination
+    assert not container_per_test.connection.file(destdir + xfilename).exists
+
+    # execution of the python module in the container
+    bci_python_wget = container_per_test.connection.run_expect(
+        [0], f"timeout 240s python3 {appdir + appl2} {url} {destdir}"
+    ).stdout
+
+    # run the test in the container and check expected keyword from the module
+    assert "PASS" in bci_python_wget
+
+    # check expected file present in the bci destination
+    assert container_per_test.connection.file(destdir + xfilename).exists
+
+
+@pytest.mark.parametrize(
+    "container_per_test", CONTAINER_IMAGES_T, indirect=["container_per_test"]
+)
+def test_tensorf(container_per_test):
+    """Test the python tensorflow library can be used for ML calculations"""
+
+    # commands for tests using python modules in the container, copied from local
+    py_tf_vers = 'python3 -c "import tensorflow as tf; print (tf.__version__)" 2>&1|tail -1;'
+
+    py_tf_test = "timeout 240s python3 " + appdir + appl1
+
+    # check the test python module is present in the container
+    assert container_per_test.connection.file(bcdir + appdir + appl1).is_file
+
+    # check the expected CPU flag for TF is available in the system
+    flg = container_per_test.connection.run_expect(
+        [0], 'lscpu| grep -i " SSE4"'
+    ).stdout
```

Comment:
The variable is not used anywhere.
```suggestion
    container_per_test.connection.run_expect(
        [0], 'lscpu| grep -i " SSE4"'
    )
```